platform/linux/do_debian_package.sh

```diff
@@ -75,7 +75,7 @@ mkdir -p "${BASE_DIR}/DEBIAN"
 cat  >"${BASE_DIR}/DEBIAN/control" <<EOF
 Section: graphics
 Priority: optional
-Depends: libsdl2-2.0-0, fonts-noto-hinted, fonts-droid-fallback, libc6 (>= 2.2.3)
+Depends: libsdl2-2.0-0, fonts-noto-hinted, fonts-droid-fallback, fonts-freefont-ttf, libc6 (>= 2.2.3)
```

Comment:
> libc6 (>= 2.2.3)

This was true at some point. Not sure what's the glibc version of the current build machine image. Maybe it needs a bump?

Reply:
```
ii  libc6:amd64    2.31-0ubuntu9.15 amd64        GNU C Library: Shared libraries
```

Reply:
good, same in [bullseye](https://packages.debian.org/es/bullseye/libc6)